sdk/Trace/NoopSpan.php

```diff
@@ -4,17 +4,21 @@
 
 namespace OpenTelemetry\Sdk\Trace;
 
-use OpenTelemetry\Context\ContextKey;
-use OpenTelemetry\Context\ContextValueTrait;
+use OpenTelemetry\Context\Context;
+use OpenTelemetry\Context\Scope;
 use OpenTelemetry\Sdk\InstrumentationLibrary;
 use OpenTelemetry\Sdk\Resource\ResourceInfo;
 use OpenTelemetry\Trace as API;
 use Throwable;
 
-class NoopSpan implements API\Span, ReadableSpan
+/**
+ * @see https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/api.md#wrapping-a-spancontext-in-a-span
+ * @todo: Implement this on the API side.
+ * @todo: Can we just use {@see https://www.php.net/manual/en/language.oop5.anonymous.php}?
+ * @todo: If not rename this to `NonRecordingSpan`.
+ */
+class NoopSpan implements ReadWriteSpan
```

Comment:
Still some work to be done here. 

Ref: https://github.com/open-telemetry/opentelemetry-php/issues/266